src/Chord.java

```diff
@@ -0,0 +1,115 @@
+import components.queue.Queue;
+import components.queue.Queue1L;
+import components.simplewriter.SimpleWriter;
+import components.simplewriter.SimpleWriter1L;
+
+/**
+ * Allows a user to transverse through a chord.
+ *
+ * @author
+ */
+public class Chord {
+
+    /**
+     * Removes the last note in a chord.
+     *
+     * @param q
+     *
+     * @return removed
+     */
+    public static char removeLast(Queue<Character> q) {
+        q.flip();
+        char removed = q.dequeue();
+        q.flip();
+
+        return removed;
+    }
+
+    /**
+     * Gets the root node of the given chord.
+     *
+     * @param q
+     *
+     * @return root
+     */
+    public static char getRoot(Queue<Character> q) {
+        char root = q.front();
+        return root;
+    }
+
+    /**
+     * Add the user given note to the chord.
+     *
+     * @param q
+     * @param note
+     */
+    public static void addNote(Queue<Character> q, char note) {
```

Comment:
Given that you expect the root to be at the front of the queue, you probably need a precondition to tell the user this. Or, you need to sort your queue manually, so the root is always at the front. I might consider making a separate component called Note that inherits Comparable, so you can sort your queue easily.